### PR TITLE
Update nep-6.mediawiki

### LINKED ISSUE #176: Broken link in NEP-6
The link to one of the implementation examples in the NEP-6 standard is broken. Likely due to some dir name changes in the `/neo` repo recently.

Current link:
https://github.com/neo-project/neo/blob/master/src/neo/Wallets/NEP6/NEP6Wallet.cs

Should be the following:
https://github.com/neo-project/neo/blob/master/src/Neo/Wallets/NEP6/NEP6Wallet.cs

### PR DESCRIPTION
**Pull request by shargon**
Close #176

### CODE REVIEW

**Review by Jim8y — APPROVED** [review]

**Review by cschuchardt88 — APPROVED** [review]

**Comment by Jim8y** [comment]
Again, i propose to reduce the number of approvals. We can wait for longer for people to review even if we have collected sufficient review, but not necessary to require that many number of approvals.

People who dont review at all are blocking.

**Comment by shargon** [comment]
I will merge it because it fix a broken link

**Pull request merged**
